cli/azd/internal/cmd/add/add_preview.go

```diff
@@ -67,6 +67,17 @@ func Metadata(r *project.ResourceConfig) resourceMeta {
 		res.UseEnvVars = []string{
 			"AZURE_OPENAI_ENDPOINT",
 		}
+	case project.ResourceTypeStorage:
+		res.AzureResourceType = "Microsoft.Storage/storageAccounts"
+		res.UseEnvVars = []string{
+			"AZURE_STORAGE_ACCOUNT_NAME",
+			"AZURE_STORAGE_BLOB_ENDPOINT",
+		}
+
+		if modelProps, ok := r.Props.(project.StorageProps); ok && !modelProps.KeylessAuth {
+			res.UseEnvVars = append(res.UseEnvVars, "AZURE_STORAGE_ACCOUNT_KEY")
+			res.UseEnvVars = append(res.UseEnvVars, "AZURE_STORAGE_CONNECTION_STRING")
```

Comment:
there is a variadic append if that looks any better to you:

```golang
res.UseEnvVars = append(res.UseEnvVars, 
     "AZURE_STORAGE_ACCOUNT_KEY",
     "AZURE_STORAGE_CONNECTION_STRING",
)
```

Reply:
Good to know, thanks 🙂